dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryFilter.java

```diff
@@ -113,21 +113,44 @@ public boolean isOperator( QueryOperator op )
     }
 
     public String getSqlOperator()
+    {
+        return getSqlOperator( false );
+    }
+
+    /**
+     *
+     * @param isOperatorSubstitutionAllowed whether the operator should be
```

Comment:
I think it would be nice to have the full Javadoc. A general description + the return definition.